patches/net/minecraft/entity/monster/EntityGuardian.java.patch

```diff
@@ -0,0 +1,66 @@
+--- a/net/minecraft/entity/monster/EntityGuardian.java
++++ b/net/minecraft/entity/monster/EntityGuardian.java
+@@ -2,12 +2,9 @@
+ 
+ import java.util.function.Predicate;
+ import javax.annotation.Nullable;
+-import net.minecraft.entity.CreatureAttribute;
+-import net.minecraft.entity.Entity;
+-import net.minecraft.entity.EntityLivingBase;
+-import net.minecraft.entity.EntityType;
+-import net.minecraft.entity.MoverType;
+-import net.minecraft.entity.SharedMonsterAttributes;
++
++import carpet.CarpetSettings;
++import net.minecraft.entity.*;
```

Comment:
make sure imports don't get squashed especially in patches

Reply:
Oh...k will make sure about it from now on...
Lemme do a commit

Reply:
Should be fixed in latest commit